dotnet/src/Planners/Planners.Handlebars.UnitTests/Handlebars/HandlebarsPlannerTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.AI.ChatCompletion;
+using Microsoft.SemanticKernel.Planning.Handlebars;
+using Moq;
+using Xunit;
+
+namespace Microsoft.SemanticKernel.Planners.UnitTests.Handlebars;
+
+public sealed class HandlebarsPlannerTests
```

Comment:
We should consolidate these with `dotnet/src/IntegrationTests/Planners/Handlebars/HandlebarsPlannerTests.cs`

Reply:
Taken

Reply:
The IntegrationTests project is for, well, integration tests.

The tests that I added are unit tests, in their own project. That's the pattern used all over SK.

I will add integration tests in the IntegrationTests project at a later time.